src/publishers.rs

```diff
@@ -60,26 +62,61 @@ pub fn publisher_users(
     crate_name: &str,
 ) -> Result<Vec<PublisherData>> {
     let url = format!("https://crates.io/api/v1/crates/{}/owner_user", crate_name);
-    let data: UsersResponse = client.get(&url).call().into_json_deserialize()?;
-    Ok(data.users)
+    if let Some(resp) = retry_get(&url, client, 3) {
+        let data: UsersResponse = resp.into_json_deserialize()?;
+        Ok(data.users)
+    } else {
+        Err(Error::new(
+            ErrorKind::ConnectionReset,
+            "Failed to retrieve publisher users",
+        ))
+    }
 }
 
 pub fn publisher_teams(
     client: &mut RateLimitedClient,
     crate_name: &str,
 ) -> Result<Vec<PublisherData>> {
     let url = format!("https://crates.io/api/v1/crates/{}/owner_team", crate_name);
-    let data: TeamsResponse = client.get(&url).call().into_json_deserialize()?;
-    Ok(data.teams)
+    if let Some(resp) = retry_get(&url, client, 3) {
+        let data: TeamsResponse = resp.into_json_deserialize()?;
+        Ok(data.teams)
+    } else {
+        Err(Error::new(
+            ErrorKind::ConnectionReset,
```

Comment:
This is somewhat misleading, since we don't know what error happened exactly. We can actually expose the real error to the user, see the docs on [`ureq::Response`](https://docs.rs/ureq/1.5.4/ureq/struct.Response.html)